### ISSUE TITLE
MediaInfo Export Options & Metdata Options

### ISSUE THREAD

**Issue by harrypm** [staxrip/staxrip#1039]
As noted on the [Discord](https://discord.com/channels/850510460256387076/850732854744973343/1126325962217377846)

The abbility to export the orignal metadata contents of files is highly useful for batched tasks, such as taking misc media and re-encoding or muxing `.mp4` to `.mkv` for example, metadata can be lost, from colour space data to mov atom data, camara data time, notes and critically timecode data.

Example: Digital tapes with sub-code timecode wont be copyed over from a `.dv` to a `.mov`, timecode will have to be taken from mediainfo dump data to copy over with a external editor like [amcdx](https://mogurenko.com/2021/01/29/amcdx-video-patcher-v0-6-7/) and manually enter it.

- [ ] Dump File on ingest
- [ ] Dump File on encode sucesss
- [ ] Dump Orignal/New file on encode sucesss

**Comment by anthony75smith75** [comment]
Yes, PLEASE can we have original meta data retained! Drives me crazy that even simple meta data like "Artist" on a music video MKV is lost. MKVToolNix retains all the meta data and MP3Tag allows us to copy it across, but this is a clumsy extra process that makes batch processing any number of files very difficult. Seems we have the tag options in "Container Configuration" in StaxRip already. For some reason the actual tag data isn't being extracted and saved in our extraction folder. Could we actually extract the tag data from the source and then write it back into the new file, all while respecting "Container Configuration" settings?

**Comment by Dendraspis** [comment]
@anthony75smith75 
Can you provide a sample file so I can see what is not transferred?

**Comment by anthony75smith75** [comment]
Here is an example:
https://drive.google.com/file/d/184OW1xt9b2J5i8YhTy0mVKql0iiEMN-f/view?usp=sharing

Use MP3 Tag and verify existent tags such as:    Title, Artist, Year, Comment, PURL, and JPG Cover.
Run through Stax Rip on Copy/Mux. We are left with only the JPG Cover, all other basic tags have been removed. If we drop the file into MKVToolnix we can also see the tags listed in another way. If we remux the file and leave the tags ticked they are copied to the new file.
